frontend/src/utils/ropeApi.ts

```diff
@@ -5,6 +5,18 @@ export interface User {
   id: number
 }
 
+export interface MoodleSettings {
+  id?: number
+  name: string
+  value: string
+}
+
+export interface SchoolDistrict {
+  id: number
```

Comment:
This should be optional similar to `MoodleSettings`. On a `POST`, we can't provide this field since it will get created by the backend on insert.